Installer/package.txt

```diff
@@ -83,11 +83,15 @@ installer LiveCode
 		desktop application "repo:Installer/application.desktop" as "runrev-[[ProductTag]]-[[TargetArchitecture]]"
 		if TargetEdition is Community then
 			desktop icon "repo:Installer/application.png" as "runrev-[[ProductTag]]-[[TargetArchitecture]]"
+		if TargetEdition is CommunityPlus then
```

Comment:
I think all of these can be reduced down to:
```
if BaseEdition is Commercial then
         desktop icon "private:engine/rsrc/[[TargetEdition]]/android-notify-icon.png" as "runrev-[[ProductTag]]-[[TargetArchitecture]]"
```
But I'm hoping to refactor this stuff so there's only really a couple of lines related to commercial editions in this file so perhaps just leave like this for now?